mobile/src/main/res/layout/widgetlist_data_saver.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/data_saver"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_vertical"
+    android:padding="16dp"
+    android:orientation="vertical"
+    android:visibility="gone"
+    tools:visibility="visible">
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/watermark"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:src="@drawable/ic_data_usage_black_64dp"
+        app:tint="@color/empty_list_text_color"
+        app:tintMode="src_in" />
+
+    <TextView
+        android:id="@+id/data_saver_hint"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:padding="16dp"
+        android:textAppearance="?android:attr/textAppearanceMedium"
+        android:textColor="@color/empty_list_text_color"
+        tools:text="Webcam is not loaded to save data" />
```

Comment:
This looks good in general, I'd probably make it a tad smaller though. Maybe 48dp for the image size, 8dp vertical padding and `?attr/textAppearanceBody2` as text appearance?

Reply:
Done